pkg/system/filesys_windows.go

```diff
@@ -80,3 +81,123 @@ func IsAbs(path string) bool {
 	}
 	return true
 }
+
+// The functions below here are extracted from the golang OS package.
```

Comment:
Might want to comment that those functions prefixed by `syscall` are extracted from the syscall package.